gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GHFSStatistic.java

```diff
@@ -0,0 +1,98 @@
+package com.google.cloud.hadoop.fs.gcs;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.hadoop.classification.InterfaceStability;
+import com.google.cloud.hadoop.fs.gcs.GHFSStatisticTypeEnum;
+import org.apache.hadoop.fs.statistics.StoreStatisticNames;
+import org.apache.hadoop.fs.statistics.StreamStatisticNames;
+
+import static com.google.cloud.hadoop.fs.gcs.GHFSStatisticTypeEnum.TYPE_COUNTER;
+import static com.google.cloud.hadoop.fs.gcs.GHFSStatisticTypeEnum.TYPE_DURATION;
+import static com.google.cloud.hadoop.fs.gcs.GHFSStatisticTypeEnum.TYPE_GAUGE;
+import static com.google.cloud.hadoop.fs.gcs.GHFSStatisticTypeEnum.TYPE_QUANTILE;
+
+
+@InterfaceStability.Unstable
+public enum GHFSStatistic {
```

Comment:
Make this class package-private if possible

Reply:
We cannot as we are using for delegation Tokens also